R/Model-class.R

```diff
@@ -324,7 +370,7 @@ LogisticLogNormal <- function(mean,
     datamodel = jags_model_data,
     priormodel = jags_model_prior,
     modelspecs = function() {
-      list(refDose = refDose, cov = cov, mean = mean)
+      list(refDose = refDose, mean = mean, cov = cov)
```

Comment:
hm not sure if this works - because we want to pass the precision matrix here - so that jags does not need to calculate it

Reply:
Yap - that wont work for sure. The `LogisticLogNormal` is not ready yet, I only adjusted the order for one specific reason. I only prepared `LogisticNormal`